app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -108,7 +108,7 @@ function defineAdminSection(
     }
 
     get name() {
-      return `admin-nav-section-${this.adminNavSectionData.name}`;
+      return `${adminSidebarStateManager.sidebarState.currentPanel.key}-${this.adminNavSectionData.name}`;
```

Comment:
This seems a little odd; since this is admin only can we just do `${ADMIN_PANEL}-${this.adminNavSectionData.name}`?

Reply:
It's mainly odd because we are calling a service from a service, which will break if the `adminSidebarStateManager` no longer relies on `sidebarState`, but it's also odd because this is only for admins so doesn't need to be dynamic